wellregistry/registry/admin/monitoring_location.py

```diff
@@ -2,7 +2,7 @@
 MonitoringLocation Admin
 """
 import csv
-
+import re
```

Comment:
it might be nice to have the separation line back

Reply:
fixed